tasks/compile-example.js

```diff
@@ -57,14 +57,21 @@ module.exports = function(templateRoot, template) {
       const document = DocumentParser.parse(contents);
       const stream = this;
       const title = FileName.toString(file);
+      let next = FileName.nextFile(file, 'src');
+      if (next) {
+        next = FileName.toString(next) + '.html';
```

Comment:
I'd prefer to fix this in toString in that case. It should simply return an empty string if next is undefined.